ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -581,38 +586,35 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
 int
 ompi_mtl_ofi_finalize(struct mca_mtl_base_module_t *mtl)
 {
+    ssize_t ret;
+
     opal_progress_unregister(ompi_mtl_ofi_progress_no_inline);
 
-    /**
- *      * Close all the OFI objects
- *           */
-    if (fi_close((fid_t)ompi_mtl_ofi.ep)) {
-        opal_output(ompi_mtl_base_framework.framework_output,
-                "fi_close failed: %s", strerror(errno));
-        abort();
-    }
-    if (fi_close((fid_t)ompi_mtl_ofi.cq)) {
-        opal_output(ompi_mtl_base_framework.framework_output,
-                "fi_close failed: %s", strerror(errno));
-        abort();
-    }
-    if (fi_close((fid_t)ompi_mtl_ofi.av)) {
-        opal_output(ompi_mtl_base_framework.framework_output,
-                "fi_close failed: %s", strerror(errno));
-        abort();
-    }
-    if (fi_close((fid_t)ompi_mtl_ofi.domain)) {
-        opal_output(ompi_mtl_base_framework.framework_output,
-                "fi_close failed: %s", strerror(errno));
-        abort();
-    }
-    if (fi_close((fid_t)ompi_mtl_ofi.fabric)) {
-        opal_output(ompi_mtl_base_framework.framework_output,
-                "fi_close failed: %s", strerror(errno));
-        abort();
-    }
+    /* Close all the OFI objects */
+    if (ret = fi_close((fid_t)ompi_mtl_ofi.ep))
+        goto finalize_err;
```

Comment:
Please use `{}` even around 1-line blocks.  It's the Open MPI Way.  😄 

(we only have a few style things that we try to be consistent about -- that's one of them)

Reply:
Sure, will fix.